src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -554,6 +557,18 @@ bool fuse_type_to_unique_v10(const std::shared_ptr<Node>& node, const precisions
     return res;
 }
 
+bool fuse_type_to_serach_sorted_v15(const std::shared_ptr<Node>& node, const precisions_map& precisions) {
+    bool res = false;
+    if (auto unique = ov::as_type_ptr<ov::op::v15::SearchSorted>(node)) {
+        auto it = precisions.find(node->get_output_element_type(0));
+        if (it != precisions.end()) {
+            unique->set_output_element_type(it->second);
+            res = true;
+        }
+    }
+    return res;
```

Comment:
The name of the casted variable should probably be something like '_search_sorted_' or '_casted_node_', but definitely not '_unique_' :)

Reply:
Fixed.